attributes/defaults.rb

```diff
@@ -18,17 +18,23 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-# Attribute that defines whether MariaDB or MySQL should be used
-default['mysqld']['use_mariadb'] = true
+# Attribute that defines whether MariaDB, MySQL or Percona should be used
+default['mysqld']['db_install'] = 'mariadb'
```

Comment:
Ok.

Percona 5.6 is included on ubuntu 16.04, but it will work since we require mysql 5.7 at least. So, `include_recipe 'mysqld::percona_repository'` needs to be included, but its a hassle since you already configured an attribute to use percona.

The same happens with mariadb, to ensure the version we want, we must include mariadb.org repositories to avoid this failing if you select version 10.2 and using default xenial repositories.